src/Environments.jl

```diff
@@ -0,0 +1,24 @@
+# Contains the structs that define the static and dynamic environment that configures/forces the river model.
+export StaticEnvironment, DynamicEnvironment, Environment
+
+struct StaticEnvironment
+    # static data objects
+    routing_lv_basins_dir::String
+    attributes_dir::String
+end
+
+struct DynamicEnvironment
+    # timeseries objects
+    # date/time - index map
+    timeseries_dir::String
```

Comment:
Maybe we could rename this "forcing_timeseries" instead of just "timeseries"?

We should add field names and doc strings for these structs as well